### PR TITLE
Add a CLI for running outside of EMR

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This PR adds a CLI for running the module directly as an egg. The runner would look like the following:

```
from mozaggregator.cli import entry_point; entry_point()
```

### CODE REVIEW

**Comment by acmiyaguchi** [comment]
The job is tested directly against databricks (which doens't pull in the pinned requirements unfortunately).

link: https://dbc-caf9527b-e073.cloud.databricks.com/#job/4210/run/1

```bash
wget https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-databricks.py
chmod +x mozetl-databricks.py

export DB_TOKEN=...
./mozetl-databricks.py \
	--git-path https://github.com/mozilla/python_mozaggregator.git \
	--git-branch cli \
	--num-workers 5 \
	--token $DB_TOKEN \
	--module-name mozaggregator \
	mobile \
	--output s3://telemetry-test-bucket/mozaggregator/mobile_test/v1/ \
	--num-partitions 200

File: mozaggregator/mobile.py [hunk 1]
```diff
@@ -179,4 +176,7 @@ def aggregate_metrics(sc, begin, end=None, num_partitions=10000):
 def run(sparkSession, begin, end=None):
     agg_metrics = aggregate_metrics(sparkSession.sparkContext, begin, end)
     aggs = get_aggregates_dataframe(sparkSession, agg_metrics)
-    write_parquet(aggs)
+    path = 's3://{bucket}/{prefix}/{version}'.format(
```
Comment (fbertsch) [inline mozaggregator/mobile.py]: Can we remove the `run` command altogether?

File: tests/test_db.py [hunk 2]
```diff
@@ -126,3 +126,57 @@ def test_notice_logging_cursor():
...
+        def from_source(*args, **kwargs):
+            return Dataset()
+
+        def where(self, *args, **kwargs):
+            return self
+
+        def records(self, *args, **kwargs):
+            return spark.sparkContext.parallelize(generate_pings())
+
+    monkeypatch.setattr("mozaggregator.aggregator.Dataset", Dataset)
```
Comment (fbertsch) [inline tests/test_db.py]: Love it

File: mozaggregator/parquet.py [hunk 3]
```diff
@@ -217,6 +215,6 @@ def aggregate_metrics(sc, channels, submission_date, main_ping_fraction=1,
                         datetime.timedelta(days=1)).strftime('%Y%m%d')
 
     aggs = aggregate_metrics(sparkSession.sparkContext, 'nightly', process_date)
-    write_aggregates(sparkSession, aggs, 'append')
+    write_aggregates(sparkSession, aggs, DEFAULT_PATH, 'append')
```
Comment (fbertsch) [inline mozaggregator/parquet.py]: Let's also remove this, if the CLI provides the same capabilities

File: mozaggregator/mobile.py (hunk 1 above)
Comment (acmiyaguchi) [reply mozaggregator/mobile.py]: Yes, although this should wait until the airflow runners are removed via https://github.com/mozilla/telemetry-airflow/pull/593. I'll put in another PR to clean-up after these ad-hoc entry points.

**Pull request merged**